src/screens/poa-form/index.js

```diff
@@ -109,14 +109,25 @@ class PoAForm extends Component<PoAFormProps, PoAFormState> {
 
   validateAddress = address => {
     const isEmpty = value => (value && value.length === 0) || !value
-
-    return {
-      // 5 digit postal codes only for now, though there is a valid 10 digit
-      // format (e.g. 12345-4321).
-      postal_code:
-        !isEmpty(address.postal_code) &&
-        !Regex.postalCode.test(address.postal_code)
-    }
+    let areAllFieldsEmpty = true
```

Comment:
This is good. Do we need to update the step validation also? It is on [line 195](https://github.com/code-for-nashville/power-of-attorney/blob/282cab31fc4729180faf25d732e56d2213d4b0cf/src/screens/poa-form/index.js#L195)

Reply:
I don't think so - do we? The validateAddress() function is bound to the mother address, father address, and the caregiver address objects. Should there be other changes?

Reply:
Ahh I see this PR is for issue #97 I was confusing it with a different issue.